pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -191,6 +191,27 @@ func (r *ReconcilePublishingStrategy) Reconcile(ctx context.Context, request rec
 		// This generated spec will be compared against the actual spec as desrcibed above
 		desiredIngressController := generateIngressController(ingressDefinition)
 
+		cloudPlatform, err := baseutils.GetPlatformType(r.client)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+		isAWS := *cloudPlatform == "AWS"
```

Comment:
just for knowledge: will NLB/classic lb types also be implemented to gcp? or is it an AWS-specific thing...?

Reply:
The request is just for AWS

Reply:
FYI I've made this, it should sort those out when time comes https://issues.redhat.com/browse/OSD-11933